tests/Http/RequestBuilderTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace YotiTest\Http;
+
+use YotiTest\TestCase;
+use Yoti\Http\AbstractRequestHandler;
+use Yoti\Http\RequestBuilder;
+use Yoti\Util\PemFile;
+
+/**
+ * @coversDefaultClass \Yoti\Http\RequestBuilder
+ */
+class RequestBuilderTest extends TestCase
+{
+    /**
+     * Test Base URL.
+     */
+    const BASE_URL = 'http://www.example.com/api/v1';
+
+    /**
+     * @var \Yoti\Util\PemFile
+     */
+    private $pemFile;
+
+    /**
+     * Setup tests.
+     */
+    public function setup()
+    {
+        $this->pemFile = $this->getMockBuilder(PemFile::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['__toString'])
+            ->getMock();
+
+        $this->pemFile->method('__toString')
+            ->willReturn(file_get_contents(PEM_FILE));
+    }
+
+    /**
+     * @covers ::build
+     */
+    public function testBuild()
+    {
+        $requestHandler = (new RequestBuilder)
+          ->withBaseUrl(self::BASE_URL)
+          ->withPemFile($this->pemFile)
+          ->withSdkIdentifier('PHP')
+          ->withSdkVersion('1.2.3')
+          ->build();
+
+        $this->assertInstanceOf(AbstractRequestHandler::class, $requestHandler);
+    }
+
+    /**
+     * @covers ::build
+     *
+     * @expectedException \Yoti\Exception\RequestException
+     * @expectedExceptionMessage Wrong Yoti SDK identifier provided: Invalid SDK
```

Comment:
If we updated the error then obvs would need to update here 👍

Reply:
Fixed in 2a8784b